.github/workflows/bundling-tests.yml

```diff
@@ -0,0 +1,16 @@
+name: Bundling tests
+
+on: pull_request
+
+permissions: read-all
+
+jobs:
+  trigger-circleci:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Start CircleCI bundling tests
+        uses: CircleCI-Public/trigger-circleci-pipeline-action@v1.0.6
+        env:
+          CCI_TOKEN: ${{ secrets.CIRCLECI_TOKEN }}
+        with:
+          GHA_Meta: 'workflow:bundling'
```

Comment:
What's the purpose of this GitHub action?

Reply:
I was trying out different things. I was working on an action that would trigger a CircleCI build only on some branches (`release/*`, not enabled here right now). But I learned this isn't possible as there's no access to secrets when running actions triggered from PRs. I'll look for another approach.

Reply:
It might be possible to configure it with CircleCI yaml.

Reply:
AFAIK CircleCI doesn't know the original branch name the PR is coming from